darknet_ros/launch/demo-v4-csp.launch.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
@Ar-Ray-code these launch files are extremely similar. Try to use one launch file and arguments instead if possible. That reduces the amount of repetitive code.

Reply:
OK.
I made the launch file simpler and reduced the number of files.
Also, demo_*.launch.py has been removed.

Reply:
And I already rebased this.